### PR TITLE
Improve the docker documentation

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: docs/src/main/resources/templates/docker.html [hunk 1]
```diff
@@ -14,11 +14,18 @@ <h4 id="docker_image">Image <a class="headerlink" href="#docker_image" title="Pe
...
     <li><code>DEFAULT_LOG_LEVEL</code>: Set the default log level</li>
     <li><code>TOMCAT_LOG_LEVEL</code>: Set the tomcat log level</li>
+    <li><code>LOG_LEVEL</code>: Set the mapfish print log level</li>
+    <li><code>SPRING_LOG_LEVEL</code>: Set the Spring log level</li>
+    <li><code>JASPER_LOG_LEVEL</code>: Set the JasperReports log level</li>
+    <li><code>APACHE_LOG_LEVEL</code>: Set the Apache log level</li>
+    <li><code>SQL_LOG_LEVEL</code>: Set the SQL (Hibernate) log level</li>
     <li><code>SENTRY_LOG_LEVEL</code>: Set the Sentry log level</li>
     <li><code>SENTRY_REPORTING_LOG_LEVEL</code>: Starting from what log level to report to Sentry.</li>
     <li><code>TOMCAT_LOG_TYPE:</code>:
```
Comment (sbrunner) [inline docs/src/main/resources/templates/docker.html]: Can we add the default values?

Comment (pvalsecc) [reply docs/src/main/resources/templates/docker.html]: Done

**Comment by jwkaltz** [comment]
This also partially addresses https://github.com/mapfish/mapfish-print/issues/601, right?

**Comment by pvalsecc** [comment]
> This also partially addresses #601, right?

Yes

File: docs/src/main/resources/templates/docker.html (hunk 1 above)
Comment (sbrunner) [reply docs/src/main/resources/templates/docker.html]: Thanks :-)

**Pull request merged**